test/io.jl

```diff
@@ -327,7 +327,19 @@ module TestIO
     io = IOBuffer(abnormal*",%_B*\tC*,end\n1,2,3\n")
     @test names(readtable(io)) == ns
 
-    # Test writetable with append
+    # Test writetable with NA
+	# Can't check the result with ==.  Also unclear how to
+	# test the results.  
```

Comment:
`readcsv` in base doesn't know about `NA`s, and should fit with tempfile approch followed below:
e.g.
```julia
df = DataFrame(A = @data([1,NA]), B = @data(["b", NA]))
writetable(tf, df)
@test readcsv(tf) == ["A" "B"; 1 "b"; "NA" "NA"]

writetable(tf, df, nastring="NaN")
@test readcsv(tf) == ["A" "B"; 1 "b"; "NaN" "NaN"]
```